libraries/AP_Camera/examples/send_camera_information.py

```diff
@@ -0,0 +1,106 @@
+import threading
+import time
+from pymavlink import mavutil
+
+ONBOARD_CONTROLLER_SYSID = 245
+ONBOARD_CONTROLLER_COMPID = 0
+
+# Function to connect to MAVLink
+def connect_to_mavlink(ip, port):
+    connection = mavutil.mavlink_connection(f'udp:{ip}:{port}', source_system=ONBOARD_CONTROLLER_SYSID)
```

Comment:
Wait for a heartbeat, lock onto the first system ID you see that is probably a vehicle heartbeat (see mavutil's `probably_vehicle_heartbeat` method)